docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -6,26 +6,70 @@ title: prometheus.scrape
 
 # prometheus.scrape
 
-`prometheus.scrape` configures a metrics scraping job for a given set of
+`prometheus.scrape` configures a Prometheus scraping job for a given set of
 `targets`. The scraped metrics are forwarded to the list of receivers passed in
 `forward_to`.
 
 Multiple `prometheus.scrape` components can be specified by giving them
 different labels.
 
+## Scraping behavior
```

Comment:
It might be better to move this section to be _after_ the example, potentially even after arguments and exports. Ditto for the intro section in #2133. 

Reference: https://developers.google.com/style/api-reference-comments?hl=en

> On each unique page (for a class, interface, etc.), include a code sample (~5-20 lines) at the top.

It feels like we'll want to keep information users will access the most frequently closest to the top (brief description, example, arguments, exports) and less frequently accessed information closer to the bottom of the page (behavior). 

cc @karengermond if you have any thoughts on this

Reply:
I'm moving both the example and the "Scraping Behavior" sections to the very bottom

I also think it ~will help~ helps with the uniformity of the reference pages; all should list arguments, exports, component behavior and then any long-winded explanations of how it works under the hood.

Reply:
@rfratto , @tpaschalis, yeah, in the old days, you'd provide a short description, usage (the command syntax), arguments, then an example if needed.  Moving this example to the bottom won't work, though, because we do need to show the usage before the arguments -- users need to see how the arguments fit/work together.  And I really like how this flows. (haha, sorry)  So maybe we should just change "Example" to "Usage"?

Reply:
Yeah, I don't believe the suggestion was to move the example to the bottom either, just the more detailed explanation that came with it. We want to enable users to copy-paste examples to get up and running as soon as possible. Not sure about the section header, no strong opinions on that ^^

Reply:
"Example" is fine.  I'm wondering whether people will find their way to the bottom to learn about the behavior. Do you think we should have a link at the bottom of the Example section to the Behavior section?  "For more details, see Scraping behavior"?

Reply:
Okay, I'll add an anchor link to that section.